### PR TITLE
refs task #5653 Create migrations

### PR DESCRIPTION
**Pull request by buihanh2304**
Create migrations
https://edu-redmine.framgia.vn/issues/5653

### CODE REVIEW

**Comment by buihanh-fr** [comment]
https://edu-redmine.framgia.vn/issues/5653

**Comment by buihanh-fr** [comment]
ready

File: database/migrations/2019_01_16_024646_create_notifications_table.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+     */
+    public function up()
+    {
+        Schema::create('notifications', function (Blueprint $table) {
+            $table->increments('id');
+            $table->unsignedInteger('user_id');
+            $table->unsignedInteger('notifier_id');
+            $table->morphs('notifiable');
+            $table->string('action');
+            $table->json('data');
```
Comment (BunnyPi04) [inline database/migrations/2019_01_16_024646_create_notifications_table.php]: vẫn dùng json à?

**Comment by buihanh-fr** [comment]
ready

**Pull request merged**